File: javascript/Q3.js
// 3. Write a function that takes three sides of a triangle and checks whether the triangle is equilateral, isosceles, or scalene.
function tri(a, b, c){
    if(a==b&&b==c){
        console.log("equilateral")
    }else if(a==b||b==c){
        console.log("isoscale")
    }else{
        console.log("scalen")
    }
}
tri(12,12,68)

File: c lang/Q3.c
// 3. Write a function that takes three sides of a triangle and checks whether the triangle is equilateral, isosceles, or scalene.


# include <stdio.h>
int main(){
	int  a,b,c;
	printf("enter the numbet");
	scanf("%d",&a);
	scanf("%d",&b);
	scanf("%d",&c);
	if(a==b&&b==c){
		printf("equilateral");
	}else if(a==b||b==c){
		printf("isoscale");
	}else{
		printf("scalar");
	}
	
}






















